_includes/plugins/katello/sidebar_3.11.html

```diff
@@ -0,0 +1,180 @@
+<ul class="header nav nav-list bs-docs-sidenav">
+  <li class="item">
+    <a>1. Installation</a>
+  </li>
+
+  <ul class="sub-header-static nav nav-list bs-docs-sidenav">
+    <li class="item">
+      <a href="/plugins/katello/3.11/installation/index.html">1.1 Katello</a>
```

Comment:
Every use of `3.11`: can it be `{{ page.version }}` as well?

Reply:
fixed for 3.11 and nightly